package.json

```diff
@@ -60,7 +62,9 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js && node scripts/buildSass.js",
-    "test": "standard && node scripts/test.js --env=jsdom"
+    "test": "standard && node scripts/test.js --env=jsdom",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
I added these scripts in my PR as well, so we will likely have a merge conflict 😢 Let's go ahead and merge this in and I'll clean up my branch.